src/Stryker.CLI/Stryker.CLI.UnitTest/Stryker.CLI.UnitTest.csproj

```diff
@@ -7,6 +7,7 @@
 
   <PropertyGroup>
     <xunit>2.4.1</xunit>
+    <LangVersion>8.0</LangVersion>
```

Comment:
The LangVersion is set in our Directory.Build.Props. c#8 is default for netcore 3.1 so removing the langversion from the directory.build.props should just work